OktaLogger.podspec

```diff
@@ -13,7 +13,6 @@ Pod::Spec.new do |s|
   s.swift_version = '5.0'
   s.dependency 'SwiftLint'
   s.default_subspec = "Complete"
-  s.static_framework = true
```

Comment:
@IldarAbdullin-okta  can't we use the same podspec and allow this to be tru only for macOS based on platform using spec.platform = :osx ?

Or you just want to keep it clean by having separate spec

Reply:
Unfortunately cocoapods doesn't allow that. `static_framework` can be only declared on spec level, declaration on sub-spec level is not required :(

Reply:
ok got it, having different spec for now is fine.